test/SlowTests/Issues/RavenDB_8328.cs

```diff
@@ -14,16 +15,14 @@ public RavenDB_8328(ITestOutputHelper output) : base(output)
         {
         }
 
-        [Fact]
-        public void SpatialOnAutoIndex()
+        [RavenTheory(RavenTestCategory.Querying)]
+        [RavenData(SearchEngineMode = RavenSearchEngineMode.All)]
+        public void SpatialOnAutoIndex(Options options)
         {
             var databaseName = $"{nameof(SpatialOnAutoIndex)}-{Guid.NewGuid()}";
             var path = NewDataPath();
             using (var store = GetDocumentStore(new Options
             {
-                Path = path,
```

Comment:
you are not setting those anywhere?

2nd document store creation also needs to use options?

Reply:
Fixed and adjusted :)